reducer-todo/src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './Todo.css';
+
+const Todo = ({item, dispatch}) => {
```

Comment:
OK so you destructed them here which brings up another thing, people will care more about consistency than whether or not you destruct your props.